translate/uz.json

```diff
@@ -1,18 +1,94 @@
 {
   "region": {
     "Toshkent": 27,
+    "Samarqand": 18,
     "Andijon": 1,
```

Comment:
`ASC` bo’yicha tartiblasak ajoyib bo’lar edi

Reply:
first 12 is by famous, then alphabetic